bootstrap.yml

```diff
@@ -514,6 +515,29 @@ packages:
         environ:
           DESTDIR: '@THIS_COLLECT_DIR@'
 
+  - name: radare2
+    source:
+      subdir: '3rdparty'
+      url: 'https://codeload.github.com/radareorg/radare2/tar.gz/refs/tags/5.4.0'
+      format: 'tar.gz'
+      extract_path: 'radare2-5.4.0'
+      version: '5.4.0'
+    revision: 0
+    tools_required:
+      - host-gcc
+    pkgs_required:
+      - mlibc
+    configure:
+      - args:
+        - 'bash'
+        - '-c'
+        - 'cd @THIS_SOURCE_DIR@ ; ./configure --without-debugger --host=x86_64-vinix --with-ostype=gnulinux --prefix=/usr'
```

Comment:
For packages that need in-tree builds, like `vim`, you should first copy the whole package source to `@THIS_BUILD_DIR@` first, then build it there. See the `vim` recipe.

Reply:
like `cp -rf @THIS_SOURCE_DIR@/* .` ? i can also build with meson, but im afraid that will require some fixes to run in vinix and also will add meson+ninja+python as dependencies which is not what i want

Reply:
Yeah like that, just check the `vim` target.